mobile.js

```diff
@@ -116,4 +116,24 @@ submit.addEventListener('click', (event) => {
     return false;
   }
   return true;
+});
+
+const userInput = document.querySelectorAll('.dataEnter');
+
+let data = {
+};
+
+/// /// local storage \\\\\\
+if (localStorage.getItem('userData')) {
+  data = JSON.parse(localStorage.getItem('userData'));
+  userInput.forEach((input) => {
+    input.value = data[input.name];
+  });
+}
+
+userInput.forEach((input) => {
+  input.addEventListener('input', () => {
+    data[input.name] = input.value;
+    localStorage.setItem('userData', JSON.stringify(data));
+  });
```

Comment:
[OPTIONAL] Well done 👍🏽 but maybe try adding some checks to see if the input has any value before storing it, when I just fill one `input` the other ones get filled with `undefined` as a string. 🤒